### DISCUSSION TITLE
I want to teach a class on this tech, specifically how to train a voice model from scratch using this lib and xtts v2

### DISCUSSION THREAD

**Discussion by marktellez** [coqui-ai/Trainer#151] [General]
There is no information that I can find about training from scratch. I have multiple datasets formatted into the coqui format, and I can finetune all day long. But starting from randomly initialized weights and pretraining isnt working well. Is there anyone here (@erogol) who could advise us enough to get going? I have multiple rtx 4090s to do the training on.

some :heart: please?

**Comment by eginhard** [comment]
Only the training code for the GPT model of XTTS was open-sourced, it's not possible to train/fine-tune the other components. There's probably not much of an advantage in training the GPT from scratch.
